src/test/java/CommandsTest.java

```diff
@@ -0,0 +1,64 @@
+import org.example.commands.Command;
+import org.example.commands.Commands;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class CommandsTest {
+    @Test
+    void executionStatusShouldBeTrueAfterCommandIsExecuted() {
+        FakeCommand command = new FakeCommand();
+        FakeConsole console = new FakeConsole("");
+
+        List<Command> commandList = new ArrayList<>();
+        commandList.add(command);
+
+        Commands commands = new Commands(commandList,console);
+        commands.executeCommand("test");
+
+        Assertions.assertTrue(command.executionStatus, "Test fail.Execution status is false. Command failed");
+    }
+
+    @Test
+    void shouldWriteMessageIncorrectCommandIfEnteredWrongCommand(){
+        FakeCommand command = new FakeCommand();
+        FakeConsole console = new FakeConsole("");
+
+        List<Command> commandList = new ArrayList<>();
+        commandList.add(command);
```

Comment:
use java prepared methods to create your commands:
new Commands(List.of(fakeCommand), console)